Setup.swift not working with tuist up

I'm running Tuist from `master`, trying to follow this: [up.md](https://github.com/tuist/tuist/blob/master/docs/usage/up.md)

I've tried a minimal `Setup.swift` that looks like this:
```swift
import Foundation
import ProjectDescription

let project = Project(
  up: [
    .homebrew(packages: ["swiftlint"]),
    .homebrew(packages: ["git-lfs"])
    .homebrew(packages: ["vault"])
  ]
)
```

Which results in the following output:
```
❌ Error: Command exited with error code 1 and error: /Users/enhorn/Repos/company/ios/Tuist/Setup.swift:5:7: error: type of expression is ambiguous without more context
  up: [
      ^
```

I've tried a shot in the dark by replacing `import ProjectDescription` with `import Project`, which resulted in this instead:
```
❌ Error: Command exited with error code 1 and error: /Users/enhorn/Repos/company/ios/Tuist/Setup.swift:4:15: error: use of unresolved identifier 'Project'
let project = Project(
              ^~~~~~~
Foundation.Process:1:12: note: did you mean 'Process'?
open class Process : NSObject {
           ^
Darwin.mprotect:1:13: note: did you mean 'mprotect'?
public func mprotect(_: UnsafeMutableRawPointer!, _: Int, _: Int32) -> Int32
            ^
ApplicationServices.PMObject:1:18: note: did you mean 'PMObject'?
public typealias PMObject = UnsafeRawPointer
                 ^
CoreServices.cObject:1:12: note: did you mean 'cObject'?
public var cObject: OSType { get }
```

Perhaps it's just not ready to be tried out yet?

--- a/docs/usage/up.md
+++ b/docs/usage/up.md
@@ -16,19 +16,17 @@ It’s a tedious process that can break without you noticing it. Moreover, each 
 
 The good news is that Tuist offers a command, **tuist up** that helps you define your project dependencies and then takes care of the configuration process for you.
 
-To define your project dependencies, we need to pass an **up** attribute to our project manifest:
+To define your project dependencies, we need to create a new `Setup.swift` manifest file:
 
 ```swift
-let project = Project(
-  up: [
+
+let setup = Setup([
     .homebrew(packages: ["swiftlint"]),
     .carthage(platforms: [.iOS])
-  ]
-  // The other manifest attributes
-)
+  ])
 ```
 
-We have turned the markdown steps that we saw before into up commands in the project manifest. When you run `tuist up`, Tuist translates those declarations into actual commands that are executed in your system.
+We have turned the markdown steps that we saw before into up commands in the setup manifest. When you run `tuist up`, Tuist translates those declarations into actual commands that are executed in your system.
 
 Moreover, it assesses whether those dependencies are already met in the environment, and if they are, it skips them. For instance, if the Carthage dependencies exist and are up to date, it doesn’t run the Carthage update command.
 

--- a/docs/usage/up.md
+++ b/docs/usage/up.md
@@ -19,6 +19,7 @@ The good news is that Tuist offers a command, **tuist up** that helps you define
 To define your project dependencies, we need to create a new `Setup.swift` manifest file:
 
 ```swift
+import ProjectDescription
 
 let setup = Setup([
     .homebrew(packages: ["swiftlint"]),

